lib/commons/utils/token-list.js

```diff
@@ -1,7 +1,14 @@
-/*global axe */
-
+/* global axe */
+/**
+ * Converts space delimited token list to an Array
+ * @method tokenList
+ * @memberof axe.commons.utils
+ * @instance
+ * @param  {String} str
+ * @return {Array}
+ */
 axe.utils.tokenList = function (str) {
```

Comment:
It's out of scope for this PR, but I don't find this utility's name to be particularly intuitive. It speaks to nothing of what it does. Does it create a token list? Process one? Get one? `tokenListToArray` would be nice.